servo/connectors/kubernetes.py

```diff
@@ -2484,6 +2550,12 @@ async def inject_sidecar(
             service_port: The port to receive ingress traffic from an upstream service.
         """
 
+        if self.workload_ref_controller:
+            await self.workload_ref_controller.inject_sidecar(
+                name=name, image=image, *args, service=service, port=port, index=index, service_port=service_port
```

Comment:
Does the defintion on line 2529 also need the service_port: Optional[int] = None callout?

Reply:
The `service_port` parameter already has an explicit default value: https://github.com/opsani/servox/blob/95c449049dc4b4041037a3877225c0cdb90fd6ee/servo/connectors/kubernetes.py#L2533

Reply:
Ah, that works. LGTM